Warcraft.NET/Files/Structures/Mat3x4.cs

```diff
@@ -0,0 +1,54 @@
+using System;
```

Comment:
Remove Mat3x4 because we now have a Matrix3x4 class

Reply:
this file does not exist in my fork, i have no idea why its in there